content/module-reference/processing-modules/diffuse.md

```diff
@@ -68,7 +68,7 @@ All speeds are added (first to fourth orders), and the sums "`first order + seco
 
 ## scale
 
-Natural diffusion is supposed to happen only to the closest neighbouring coordinates. That is, at each iteration, each pixel should only interact with its 9 nearest neighours.
```

Comment:
Much as it pains me, this should use the American spelling - so "neighbors"

Reply:
Ok. Changed to "neighbors". But in this case, to be consistent I also changed the "neihbouring coordinates" to "neighboring corrdinates", to be consistent.